### PR TITLE
feat : Add tooltips to MANA component

### PR DESCRIPTION
**Pull request by Chakravarthy7102**
Screenshot (8)
Screenshot (7)

Related to https://github.com/decentraland/marketplace/issues/699

### CODE REVIEW

**Comment by Chakravarthy7102** [comment]
@kevinszuchet  here is the new PR!

**Comment by Chakravarthy7102** [comment]
> Looks great, thanks!
> 
> Would you mind following the 1st task of the [description of the issue](https://github.com/decentraland/marketplace/issues/699)? The new property should be called `showTooltip`
> 
> > Add an optional showTooltip boolean prop to the  component in [decentraland-ui](https://github.com/decentraland/ui)
> 
> Also, the tooltip should show `${network} MANA` instead of the amount. Would you mind changing it?

Done!

File: src/components/Mana/Mana.tsx [hunk 1]
```diff
@@ -46,5 +58,18 @@ export class Mana extends React.Component<ManaProps & HeaderProps> {
         {children}
       </Header>
     )
+
+    if (showTooltip) {
```
Comment (kevinszuchet) [inline src/components/Mana/Mana.tsx:62]: Just as a small comment, we usually write: `return showTooltip ? <Popup {...props} /> : ManaInner`.
Not mandatory though IMO.

**Review by kevinszuchet — APPROVED** [review]
Great work, thanks!

**Comment by Chakravarthy7102** [comment]
> Great work, thanks!

@kevinszuchet  I think now we should enable this tooltip in different places right?

**Comment by kevinszuchet** [comment]
@Chakravarthy7102 yep, that's the next step. Would you like to do that?

**Comment by Chakravarthy7102** [comment]
> @Chakravarthy7102 yep, that's the next step. Would you like to do that?

@kevinszuchet  Yess, I want to do this but please just guide me on where to update this component IM in which repositories?

**Comment by kevinszuchet** [comment]
You can start in the [marketplace repo](https://github.com/decentraland/marketplace), and you can use the new PR to close the [original issue](https://github.com/decentraland/marketplace/issues/699).

A good starting point is what the description of the issue says in its last task:
> Turn on the property in all the relevant places in the marketplace (item/nft cards, item/nft details, buy/sell/bid modals when it's not within an input)

You should be able to search for the MANA component in those places and pass the new argument that you just added 🚀 

Please let me know if I can be of help somehow.

**Pull request merged**